libs/orbit-essentials/src/install_chunked_code.rs

```diff
@@ -181,3 +183,31 @@ pub async fn install_chunked_code(
     }
     Ok(())
 }
+
+pub async fn install_chunked_code(
+    target_canister: Principal,
+    install_mode: CanisterInstallMode,
+    module: Vec<u8>,
+    module_extra_chunks: Option<WasmModuleExtraChunks>,
+    arg: Vec<u8>,
+) -> Result<(), String> {
+    let has_extra_chunks = module_extra_chunks.is_some();
+    let res = install_chunked_code_(
+        target_canister,
+        install_mode,
+        module,
+        module_extra_chunks,
+        arg,
+    )
+    .await;
+    if has_extra_chunks {
+        // clear the ICP chunk store of the target canister to reduce memory footprint
+        // and ignore any errors here so that a successful code install is not reported
+        // as failed only because of a failure to clear the ICP chunk store
+        let _ = mgmt::clear_chunk_store(ClearChunkStoreArgument {
```

Comment:
any reason to create a separate function for this call instead of adding it after `install_chunked_code`?

Reply:
I've removed the new function now: the motivation was to avoid refactoring of the existing (quite large) function to install chunked code.